justfile

```diff
@@ -0,0 +1,156 @@
+
+# Make sure that some env variables are set for all jobs.
+#FIXME: Consider using .env for this, but only if used also on CI
+export RUST_WORKSPACE := env_var_or_default("RUST_WORKSPACE", "discovery_engine_core")
+export DART_WORKSPACE := env_var_or_default("DART_WORKSPACE", "discovery_engine")
+export CARGO_INSTALL_ROOT := env_var_or_default("CARGO_INSTALL_ROOT", "cargo-installs")
+
+# Run just --list
+default:
+    @{{just_executable()}} --list
+
+# Get/update dart deps
+dart_deps:
```

Comment:
Personal note: it appears that justfiles support dashes instead of underscores in names. I find that it's easier to type `just dart-fmt` than `just dart_fmt`, and it also seems more "idiomatic" (I'm using this term very loosely here) when it comes to command line arguments, which in my experience rarely use underscores.

Reply:
I fully agree, I somehow had the misconception you can't do that. I will change it.